scripts/test

```diff
@@ -1,41 +1,37 @@
 #!/bin/bash
 set -e
 
+CWD=$(pwd -P)
 cleanup()
 {
     EXIT=$?
-    kill $PID
+    set +e
+    echo Stopping rancher server
+    kill %%
```

Comment:
This should be explicit, we can't guarantee it's rancher that is going to get the axe here

Reply:
Updated